rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/api/CoordinatorApi_1_0_IT.java

```diff
@@ -0,0 +1,831 @@
+            URI lraIdFromLocationHeader = URI.create(response.getHeaderString(HttpHeaders.LOCATION));
+            Assert.assertEquals("Expecting the LOCATION header configures the same LRA id as entity content on starting nested LRA",
+                    lraId2, lraIdFromLocationHeader);
+            // context header is returned strangely to client, some investigation will be needed
+            // String lraContextHeader = response.getHeaderString(LRA.LRA_HTTP_CONTEXT_HEADER);
+            // the context header is in format <parent LRA id>,<nested LRA id>?ParentLRA=<parent LRA id>
+            // MatcherAssert.assertThat("Expected the nested LRA context header gives the parent LRA id at first",
+            //        lraContextHeader, startsWith(lraId1.toASCIIString()));
+            // MatcherAssert.assertThat("Expected the nested LRA context header provides LRA id of started nested LRA",
+            //        lraContextHeader, containsString("," + lraId2.toASCIIString()));
+            Assert.assertEquals("Expecting to get the same API version as used for the request on nested LRA start",
+                    API_VERSION_1_0, response.getHeaderString(LRA_API_VERSION_HEADER_NAME_V1_0));
+        }
+
+        Collection<URI> returnedLraIds = lraClient.getAllLRAs().stream().map(LRAData::getLraId).collect(Collectors.toList());
+        MatcherAssert.assertThat("Expected the coordinator knows about the top-level LRA", returnedLraIds, hasItem(lraId1));
+        MatcherAssert.assertThat("Expected the coordinator knows about the nested LRA", returnedLraIds, hasItem(lraId2));
+
+        try (Response response = client.target(coordinatorUrl)
+                .path(encodedLraId1 + "/close")
+                .request()
+                .header(LRA_API_VERSION_HEADER_NAME_V1_0, API_VERSION_1_0)
+                .put(null)) {
+            lrasToAfterFinish.clear(); // we've closed the LRA manually here, skipping the @After
+            Assert.assertEquals("Closing top-level LRA should be successful, PUT/200 is expected.",
+                    Status.OK.getStatusCode(), response.getStatus());
+            Assert.assertEquals("Closing top-level LRA should return the right status.",
+                    LRAStatus.Closed.name(), response.readEntity(String.class));
+            Assert.assertEquals("Expecting to get the same API version as used for the request to close top-level LRA",
+                    API_VERSION_1_0, response.getHeaderString(LRA_API_VERSION_HEADER_NAME_V1_0));
+        }
+
+        Collection<LRAData> activeLRAsAfterClosing = lraClient.getAllLRAs().stream()
+                .filter(data -> data.getLraId().equals(lraId1) || data.getLraId().equals(lraId2)).collect(Collectors.toList());
+        MatcherAssert.assertThat("Expecting the started LRAs are not more active after closing the top-level one",
+                activeLRAsAfterClosing, emptyCollectionOf(LRAData.class));
+    }
+
+    @Test // TODO: delete me
```

Comment:
I will skip these tests with "delete me" comments for now. I can review them if needed later.